subsys/net/lib/nrf_cloud/coap/src/nrf_cloud_coap_transport.c

```diff
@@ -244,6 +244,8 @@ int nrf_cloud_coap_connect(const char * const app_ver)
 		goto fail;
 	}
 
+	/* TODO: Update shadow with control section, only on initial connect */
```

Comment:
the shadow state could also be requested here, this would allow for the `nrf_cloud_coap_shadow_default_process` to be performed, instead of doing it in `nrf_cloud_coap_shadow_delta_process`, which must be called by the user.

Reply:
Yes, I have thought about doing so, but the fact that it does return the shadow delta (not the whole state) on update is an undocumented feature of the CoAP server.  It is worth asking Szymon if that is intentional and will remain.

Reply:
removed this TODO.
updated to set the control section on initial connect.
for now, the mqtt shadow stuff will just be handled when the device processes the first delta.